Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/NestedClassWithoutPartialParent.diagnostics.cs

```diff
@@ -0,0 +1,14 @@
+﻿[
+  {
+    "Id": "RLM027",
+    "Severity": 3,
+    "Message": "Class OuterClassNonPartial contains nested Realm class OuterClassNonPartial.NestedClassWithoutPartialParent and needs to be declared as partial.",
```

Comment:
Now that we have the name of the parent, we probably don't need the full name of the child.